README.md

```diff
@@ -33,6 +35,11 @@ logger = Logger.new(STDOUT)
 logger.formatter = Chitragupta::JsonLogFormatter.new
 ```
 
+Passing values for `log.*` or `meta.*`
+```
+Rails.logger.info({ log: { id: 'some-unique-id', kind: 'UNIQUE_KIND' }})
```

Comment:
```suggestion
Rails.logger.info({ 'log': { 'id': 'some-unique-id', 'kind': 'UNIQUE_KIND' }})
```

Reply:
This is a rails Hash. These are symbols in the hash. Need not wrap it in a quote. A quote is helpful with symbols when you want the symbol to have a space ` ` in the middle